cmd/minikube/cmd/root.go

```diff
@@ -233,10 +233,13 @@ func init() {
 // initConfig reads in config file and ENV variables if set.
 func initConfig() {
 	configPath := constants.ConfigFile
+	_, err := os.Stat(configPath)
+	if os.IsNotExist(err) {
+		return
```

Comment:
Is it safe to just return silently here?

Reply:
In the _other_ PR (#4940), we issued a warning